WF-712: Updated repo for new toolbelt changes (#15)
spatialos/FlexibleProjectExample: docs/reference/project-configuration.md

| `projectName`  | Required  | The name of the project assigned to you by Improbable. It's used in the Console and for deployments. It should be between 3 and 32 characters long and contain letters, numbers and underscores. |
| `schemaDescriptor` | Required | A relative (to the location of this file) or absolute path to the SpatialOS schema descriptor file. This is a binary file obtained by using the SpatialOS schema compiler on all of the project's `.schema` files. [`build.sh`](../../build.sh) provides an example on how to generate this file. <br><br> This file is required in order for the SpatialOS Runtime to understand the schema used by all the workers. |
| `clientWorkers` | Optional | A list of relative (to the location of this file) or absolute paths to client worker configuration files. See the [client worker config](client-worker-configuration.md) page for more details. |

> | `serverWorkers` | Optional | A list of relative (to the location of this file) or absolute paths to server worker configuration files. See the [server worker config](server-worker-configuration.md) page for more details. |

Reviewer: ```suggestion
| `serverWorkers` | Optional | A list of relative (to the location of this file) or absolute paths to server-worker configuration files. See the [server-worker config](server-worker-configuration.md) page for more details. |
```

Author: Done.

> | `launch_config` | Optional | Path to the default [launch configuration](launch-configuration) to start deployments with. The launch configuration to use can be overwritten using the `--launch_config` flag when starting a deployment with the spatial CLI. |

Reviewer: ```suggestion
| `launch_config` | Optional | Path to the default [launch configuration](launch-configuration) to start deployments with. You can overwrite this by using the `--launch_config` flag when starting a deployment with the spatial CLI. |
```

Author: Done.
